python/tvm/relax/frontend/onnx_frontend.py

```diff
@@ -0,0 +1,1150 @@
+                self.bb.emit_normalized(binding)
+        else:
+            final_binding = var_bindings[-1]
+            for binding in var_bindings[:-1]:
+                self.bb.emit_normalized(binding)
+
+        return final_binding.value
+
+    def _convert_operator(self, op_name, inputs, attrs, opset):
+        """Convert ONNX operator into a Relax operator.
+        The converter must specify conversions explicitly for incompatible name, and
+        apply handlers to operator attributes.
+        Parameters
+        ----------
+        op_name : str
+            Operator name, such as Convolution, FullyConnected
+        inputs : list of tvm.relax.function.Function
+            List of inputs.
+        attrs : dict
+            Dict of operator attributes
+        opset : int
+            Opset version
+        Returns
+        -------
+        sym : tvm.relax.function.Function
+            Converted relax function
+        """
+        convert_map = _get_convert_map(opset)
+        if op_name in convert_map:
+            convert_class = convert_map[op_name]
+            op_function = convert_class.get_converter(opset)
+            # If the op_function is a subclass of Relay OnnxOpConverter then it is a relay op.
+            if issubclass(convert_class, RelayOnnxOpConverter):
+                relay_inputs = self._relay_input_adapter(inputs)
+                # The op_function might change the inputs to the relay op. Use a copy of the inputs.
+                relay_inputs_copy = onnx_input()
+                for relay_input in relay_inputs:
+                    relay_inputs_copy.append(relay_input)
+                # TODO handle params passing
+                relay_output = op_function(relay_inputs_copy, attrs, params=[])
+                sym = self._relay_output_adapter(inputs, relay_inputs, relay_output)
+            else:
+                sym = op_function(self.bb, inputs, attrs)
+        else:
+            raise NotImplementedError("Operator {} not implemented.".format(op_name))
+        return sym
+
+
+def from_onnx(model, shape=None, dtype="float32", opset=None, target: Union[str, Target] = "llvm"):
+    """Convert a ONNX model into an equivalent Relax Function.
+    ONNX graphs are represented as Python Protobuf objects.
+    The companion parameters will be handled automatically.
+    However, the input names from onnx graph is vague, mixing inputs and
+    network weights/bias such as "1", "2"...
+    For convenience, we rename the `real` input names to "input_0",
+    "input_1"... And renaming parameters to "param_0", "param_1"...
+    By default, ONNX defines models in terms of dynamic shapes. The ONNX importer
+    retains that dynamism upon import, and the compiler attempts to convert the
+    model into a static shapes at compile time. If this fails, there may still
+    be dynamic operations in the model. Not all TVM kernels currently support
+    dynamic shapes, please file an issue on discuss.tvm.apache.org
```

Comment:
```suggestion
    dynamic shapes, please file an issue on https://github.com/apache/tvm/issues
```

should users not be bringing bugs to GitHub instead of the forums?